Signal/src/ViewControllers/ConversationView/ConversationViewItem.h

```diff
@@ -48,6 +48,8 @@ NSString *NSStringForOWSMessageCellType(OWSMessageCellType cellType);
 // This property will be non-zero if the attachment is valid.
 @property (nonatomic, readonly) CGSize mediaSize;
 
+@property (nonatomic, readonly, nullable) NSString *caption;
```

Comment:
Note that this change is in `ConversationMediaGalleryItem`, which I might move to a separate header.